### PR TITLE
feat: view project cell count estimate (#799)

### LINKED ISSUE #799: [HCA DB] View project cell count estimate
#### Approach
* See [mocks](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/HCA-Data-Explorer?node-id=237%3A1534&t=PLF6ISGGar3bRB7v-0).
* See `ProjectViewFactory` and `table-methods.rollupMetadata` for any existing special handling for cell count estimate.

### PR DESCRIPTION
**Pull request by frano-m**
### Ticket
Closes clevercanary/data-browser#799

### Reviewers
@NoopDog 

### Changes
- Added cell count estimate to project view.

### Definition of Done (from ticket)
- Project view displays cell count estimate.

### CODE REVIEW

File: explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts [hunk 1]
```diff
@@ -645,15 +645,15 @@ export const buildTripleColumnGrid = (): React.ComponentProps<
  */
 function calculateEstimatedCellCount(
```
Comment (frano-m) [inline explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts:646]: Formatting of cell count has been removed from the function `calculateEstimatedCellCount` to `getEstimatedCellCount`.

File: explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts [hunk 2]
```diff
@@ -645,15 +645,15 @@ export const buildTripleColumnGrid = (): React.ComponentProps<
...
   const estimatedCellCount =
     getProjectResponse(projectsResponse).estimatedCellCount;
   // If there's an estimated cell count for the project, return it as the cell count.
   if (estimatedCellCount) {
-    return estimatedCellCount.toLocaleString();
+    return estimatedCellCount;
   }
   // Otherwise, return the cell suspension total count.
-  return getCellSuspensionTotalCells(projectsResponse);
+  return rollUpTotalCells(projectsResponse);
```
Comment (frano-m) [inline explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts:656]: `rollUpTotalCells` is the returned calculated value of `totalCells` in the function `getCellSuspensionTotalCells` - so essentially we have removed the formatting of this value.

File: explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts [hunk 3]
```diff
@@ -820,6 +820,26 @@ function getDCPGeneratedMatricesTableColumns<T>(): ColumnDef<T>[] {
   ];
 }
 
+/**
+ * Returns formatted estimated cell count from the given projects response.
+ * @param projectsResponse - Response model return from projects API.
+ * @param formatFn - Function to format count (optional, e.g. formatCountSize, default is locale string).
+ * @returns formatted estimated cell count.
+ */
+export function getEstimatedCellCount(
```
Comment (frano-m) [inline explorer/app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders.ts:829]: The new function `getEstimatedCellCount` formats the calculated cell count by either locale string or a specified format function like `formatCountSize`.

**Review by MillenniumFalconMechanic — APPROVED** [review]
LGTM, thanks F!

**Pull request merged**